src/i18n/fr.json

```diff
@@ -1,91 +1,76 @@
-    "no_e16d9132": "Non",
-    "ok_c47544a2": "OK",
-    "options_3ab0ea65": "Options",
-    "please_tell_us_some_information_to_start_the_chat_ac135cbb": "Veuillez nous donner quelques informations pour démarrer la conversation",
-    "please_wait_for_the_next_available_agent_b2a49c4c": "Veuillez patienter, attente d'un agent disponible...",
-    "powered_by_rocket_chat_4d7c2ab4": "Propulsé par Rocket.Chat",
-    "restore_chat_3bfecf2b": "Restaurer conversation",
-    "room_name_changed_9c42350a": "Nom du salon modifié",
-    "send_e3bd0ed0": "Envoyer",
-    "sound_is_off_a743f419": "Le son est désactivé",
-    "sound_is_on_98a9ec58": "Le son est activé",
-    "start_chat_8606d464": "Démarrer le chat",
-    "thanks_for_talking_with_us_719cce22": "Merci d'avoir conversé avec nous",
-    "the_chat_was_moved_back_to_queue_cefbade6": "The chat was moved back to queue",
-    "the_chat_was_transferred_to_another_agent_b375bdac": "The chat was transferred to another agent",
-    "the_controller_of_your_personal_data_is_company_na_c82f5567": "L'opérateur de vos données personnelles est [Company Name], dont l'adresse est [Company Address]. Pour débuter le chat vous acceptez que vos données personnelles soient utilisées et transmises conformément au règlement général sur la protection des données (RGPD).",
-    "type_your_message_here_6a05bd0f": "Tapez votre message ici",
-    "unread_messages_5e18e7b7": "messages non lus",
-    "user_added_by_525b6b11": "Utilisateur ajouté par",
-    "user_joined_407ba0d": "L'utilisateur a rejoint",
-    "user_left_58ed9c36": "L'utilisateur est parti",
-    "user_removed_by_e990f856": "Utilisateur supprimé par",
-    "waiting_queue_800061da": "File d'attente...",
-    "we_are_not_online_right_now_please_leave_a_message_57df1966": "Nous ne sommes pas en ligne en ce moment. Merci de laisser un message.",
-    "welcome_dd4e7151": "Bienvenue",
-    "write_your_message_6eee0188": "Ecrivez votre message...",
-    "yes_dde87d5": "Oui",
-    "you_browser_doesn_t_support_audio_element_3391386f": "Votre navigateur ne supporte pas les éléments audio",
-    "you_browser_doesn_t_support_video_element_e9cbd81e": "Votre navigateur ne supporte pas les éléments vidéo",
-    "your_spot_is_spot_a35cd288": "Votre rang est #%{spot}",
-    "your_spot_is_spot_estimated_wait_time_estimatedwai_d0ff46e0": "Votre rang est #%{spot} (Temps d'attente estimé: %{estimatedWaitTime})"
-  }
+    "translation": {
+        "are_you_sure_you_want_to_finish_this_chat": "Voulez-vous vraiment terminer cette conversation ?",
+        "are_you_sure_you_want_to_remove_all_of_your_person": "Voulez-vous vraiment supprimer toutes vos données personnelles ?",
+        "are_you_sure_you_want_to_switch_the_department": "Êtes-vous sûr de vouloir changer de département ?",
+        "cancel": "Annuler",
+        "change_department": "Changer de département",
+        "change_department_1": "Changer Département",
+        "chat_finished": "Chat Terminé",
```

Comment:
```suggestion
        "chat_finished": "Chat Terminé",
        "chat_started": "Chat démarré",
```

Reply:
suggesting this since it was mentioned by the support team over [here](https://open.rocket.chat/group/jwbhsdybxuewsxjcw?msg=NwnxWg32nEjH3FTWD)